src/apidefs/swagger.ts

```diff
@@ -0,0 +1,32 @@
+import * as lbOpenApiSpec from '@loopback/openapi-spec';
+import {OpenApiSpec} from '@loopback/openapi-spec';
+import {helloWorldApi} from './hello-world.api';
+
+export let spec = lbOpenApiSpec.createEmptyApiSpec();
```

Comment:
Can we use `const spec` and then `Object.assign(spec, helloWorldApi)`? Just an idea.

Reply:
Yea I had it like that at some point, but the compiler complained about it being a read only property:
```
[ts] Cannot assign to 'spec' because it is a constant or a read-only property.
```